components/common/MainNav.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+import TetherComponent from 'react-tether';
+import Router from 'next/router';
+
+// components
+import SubMenu from 'components/common/SubMenu';
+
+// constants
+import { EXPLORE_SECTIONS } from 'constants/common';
+
+export default class MainNav extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      active: ''
+    };
+  }
+
+  onSelectSection(e, active) {
+    if (e) e.preventDefault();
+    const currentActive = this.state.active;
+    if (currentActive === active) return;
+
+    this.setState({
+      active
+    });
+  }
+
+  onCloseSubMenu() {
+    this.setState({
+      active: ''
+    });
+  }
+
+  // take a look, this is not working
+  onClickItem(pathname, category) {
+    Router.push(`/${pathname}/${category}`);
```

Comment:
Use `react-routes`'s router instead. If tomorrow we change the format of the URL, it won't break.

https://github.com/fridays/next-routes#on-the-client